app/src/plugins/vuetify.ts

```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
+import Vuetify from 'vuetify'
```

Comment:
This little change meant the difference between the Vuetify components not being registered and everything working correctly. No build errors, just runtime :(